sdk/keyvault/Azure.Security.KeyVault.Administration/tests/BackupRestoreRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.RegularExpressions;
+using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Tests;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class BackupRestoreRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        /// <summary>
+        /// A RegEx to isolate a SAS token value.
+        /// </summary>
+        private Regex SignatureRegEx = new Regex(@"([\x0026|&|?]sig=)([\w\d%]+)", RegexOptions.Compiled);
+
+        public override string SanitizeVariable(string variableName, string environmentVariableValue)
+        {
+            return variableName switch
+            {
+                KeyVaultTestEnvironment.PrimaryKeyEnvironmentVariableName => string.Empty,
```

Comment:
Use `SanitzedValue` or something like that from the `base` instead so it's easier to spot in JSON.

Reply:
So this won't be in the body at all? If so, use the new sanitizer options from Azure.Core.TestFramework that @maririos just added.